content/en/nodes.md

```diff
@@ -23,6 +23,6 @@ Thus, nodes run apps or processes as part of a group forming a [cluster](/cluste
 ## How it helps
 
 A node gives you a distinct unit of compute (memory, CPU, network) that you can assign to a cluster. 
-In a cloud native platform or app a node represents a single unit that can perform work. 
+In a [cloud native](/cloud-native-apps/) platform or app a node represents a single unit that can perform work. 
```

Comment:
I would rather link this to cloud native technology than to cloud native apps, what do you think @jihoon-seo @seokho-son @CathPag ?

Reply:
@iamNoah1 
I also prefer to link to cloud native technology. (maybe related description needs to be enhanced accordingly)

Reply:
Thanks @iamNoah1 and @seokho-son I will change it now.